From 413f074d3478f290b3ebe52145963e1ad2b114d7 Mon Sep 17 00:00:00 2001
From: Roumen Petrov <local@example.net>
Date: Tue, 5 Mar 2013 23:38:02 +0200
Subject: [PATCH 17/24] MINGW: find import library

required by setup.py on hosts with installed only shared (DLL) packages
---
 Lib/distutils/cygwinccompiler.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Lib/distutils/cygwinccompiler.py b/Lib/distutils/cygwinccompiler.py
index 08c0b87..3da7b40 100644
--- a/Lib/distutils/cygwinccompiler.py
+++ b/Lib/distutils/cygwinccompiler.py
@@ -50,6 +50,7 @@ class CygwinCCompiler(UnixCCompiler):
     obj_extension = ".o"
     static_lib_extension = ".a"
     shared_lib_extension = ".dll"
+    dylib_lib_extension = ".dll.a"
     static_lib_format = "lib%s%s"
     shared_lib_format = "%s%s"
     exe_extension = ".exe"
-- 
1.7.12.1